polls/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from accounts.models import CustomUser
+
+# Create your models here.
+
+class Company(models.Model):
+    name = models.CharField(max_length=100)
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+class Poll(models.Model):
+    company = models.ForeignKey(Company, related_name='polls', on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+    price = models.IntegerField()
+    category = models.ForeignKey(Category, on_delete=models.DO_NOTHING)
+    short_description = models.TextField(max_length=50)
+    description = models.TextField(max_length=500)
+    time_needed = models.TimeField()
+    rating = models.FloatField()
+    assigned_users = models.ManyToManyField(CustomUser, related_name='assigned_polls', through='PollAssignment')
+
+class PollAssignment(models.Model):
+    user = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
+    poll = models.ForeignKey(Poll, on_delete=models.CASCADE)
+    assigned_date = models.DateTimeField()
```

Comment:
Probably its worth to use `auto_add_now` here

https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.DateField.auto_now_add